multisite-directory.php

```diff
@@ -213,37 +212,37 @@ private function initializeDirectory () {
     }
 
     /**
-     * Outputs site directory category fields during new site signup on front-end.
+     * Outputs site directory category fields during new site signup on front-end or the Network Admin screen.
      *
      * @link https://developer.wordpress.org/reference/hooks/signup_blogform/
+     *
+     * @uses $pagenow to detect if the form is on the Network Admin add new site screen and output matching HTML
      */
     public static function signup_blogform () {
+        global $pagenow;
         require_once ABSPATH.'wp-admin/includes/template.php';
         $html = wp_terms_checklist(0, array(
             'taxonomy' => Multisite_Directory_Taxonomy::name,
             'echo'     => false,
         ));
-        print '<div id="multisite-directory-signup-categories">';
-        print '<label>'.__('Site Categories:', 'multisite-directory').'</label>';
-        print str_replace("disabled='disabled'", '', $html);
-        print '</div><!-- #multisite-directory-signup-categories -->';
-    }
+        $html = str_replace("disabled='disabled'", '', $html);
+        $label = '<label>'.__('Site Categories:', 'multisite-directory').'</label>';
 
-    /**
-     * Adds the site categories to the default site creation meta variables.
-     *
-     * @link https://developer.wordpress.org/reference/hooks/add_signup_meta/
-     *
-     * @param array $meta
-     */
-/*
-    public static function add_signup_meta ($meta) {
-        if (isset($_POST['tax_input']) && !empty($_POST['tax_input'][Multisite_Directory_Taxonomy::name])) {
-            $meta['multisite-directory-signup-categories'] = $_POST['tax_input'][Multisite_Directory_Taxonomy::name];
+        print '<div id="multisite-directory-signup-categories">';
+        if ($pagenow == 'site-new.php') {
```

Comment:
Oh yes, I'm familiar with the technique but didn't know people called it "Yoda conditions". Now somehow it seems more appealing :D I just don't like the way it looks in the code, but I can understand the benefits. I also wasn't aware that it is a WP coding standard. I'll include this change in the final commit.